### PR TITLE
feat : add altlabel, uri, abbreviation, sousTelleDe & UniteDe in get/…

### PR DESCRIPTION
**Pull request by HugoBouttes**
…organisations/{id}

Issue #65

### CODE REVIEW

**Review by FBibonne — COMMENTED** [review]
Should add please test for method OrganisationsImpl.getAllOrganisations that you modified

Should rebase your branch on main because you are one commit behind main : merge could make the code fail

File: src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java [hunk 1]
```diff
@@ -41,19 +66,67 @@ public void setId(String id) {
...
+
+    public String getUniteDe() {
+        return uniteDe;
+    }
+
+    public void setUniteDe(String uniteDe) {
+        uniteDe = uniteDe;
+    }
+
+    public String getSousTelleDe() {
```
Comment (EmmanuelDemey) [inline src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java:93]: Is it an english word ?

File: src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java [hunk 2]
```diff
@@ -41,19 +66,67 @@ public void setId(String id) {
...
+        return sousTelleDe;
+    }
+
+    public void setSousTelleDe(String sousTelleDe) {
+        this.sousTelleDe = sousTelleDe;
+    }
+
+    public String getPreflabelLg2() {
```
Comment (EmmanuelDemey) [inline src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java:101]: getPrefLabelLg2 (and for the others getters and setters ;))

**Review by EmmanuelDemey — CHANGES_REQUESTED** [review]

File: src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java (hunk 1 above)
Comment (HugoBouttes) [reply src/main/java/fr/insee/rmes/model/organisation/OrganisationModel.java:93]: No a french one. In the spec they are asking for a "SousTelleDe" property which mean "under the supervision of "

**Review by EmmanuelDemey — APPROVED** [review]

**Review by JeromeMBourgeois — APPROVED** [review]

**Pull request merged**